### DISCUSSION TITLE
how do i create an index2.html so that my default index.html wails to talk to this new page?

### DISCUSSION THREAD

**Discussion by Juniornewxt** [wailsapp/wails#2758] [General]
how do i create an index2.html so that my default index.html wails to talk to this new page? I created inside the my-element.js file the path "<"a id="Index2" href="index2.html">Link</a>"
and inside the index2.html I created a new file "<script src="./src/second-element.js" type="module"></script>"
when I'm running "wails dev" the link to index2 works, but no button inside second-element.js works and if I compile the program to "wails build" then the link doesn't work, how do I solve this ?

If anyone has an example of how to do it will help me a lot, thanks.

**Comment by roma-marshall** [comment]
@Juniornewxt If you use Apache, you can try it: https://ubiq.co/tech-blog/how-to-change-default-index-page-in-apache/

**Comment by Juniornewxt** [comment]
I solved this by creating other my-elements
